test/contact-forms.spec.js

```diff
@@ -128,6 +128,28 @@ describe('contact forms', () => {
     expect(result.errors).to.be.empty;
   });
 
+  it('msf-goma edit person', async () => {
```

Comment:
can you add a test which creates a contact, edits it without any inputs (just use all default values), and confirm that the contact is unchanged before and after the edit? is this possible?

Reply:
With an edit contact form that doesn't have required inputs, it is possible